### ISSUE TITLE
Social Media Guidelines Need To Meet Accessibility Standards

### ISSUE THREAD

**Issue by quantumpacket** [GreenPartyUS/social-media#1]
`social_media_guidelines.html` needs to be made WCAG 2.0 AAA compliant.

Currently:
 - contrast ratios for background colors do not meet even AA rating.
 - missing ARIA attributes for navigation and item collapse/expand functions
 - not screen reader friendly
  - does not differentiate between do/don't items
